### DISCUSSION TITLE
question regarding `~` named directory

### DISCUSSION THREAD

**Discussion by Ioulan** [marlonrichert/zsh-autocomplete#529] [Q&A] [answered]
Sorry If this is not somehow related to zsh-autocomplete plugin. 

As per the image below when I type tilde `~`It does give me or suggesting me a named-directory and which is your plugin name and when I pressed enter It does cd to that directory
![Screenshot from 2022-12-15 17-09-25](https://user-images.githubusercontent.com/114978689/208646239-0372246a-af97-4238-a7ea-e417e0bdf9e5.png)

Also, this when updating your plugin it is using `~` and git pulling even though I'm not on a git directory
`git -C ~zsh-autocomplete pull`

Now my question Is how did you that? How does that thing works? (I want to add my custom directory too)

**Comment by yutkat** [comment, marked as answer]
You can use `hash` command to do that.

Use it like this.
```zsh
hash -d zshdata=$XDG_DATA_HOME/zsh
```

See the document for details.
```
hash [ -Ldfmrv ] [ name[=value] ] ...
```
https://zsh.sourceforge.io/Doc/Release/Shell-Builtin-Commands.html

**Reply by Ioulan** [reply]
Thank you very much!!
